server/src/main/java/org/elasticsearch/action/admin/indices/close/CloseIndexResponse.java

```diff
@@ -18,20 +18,44 @@
  */
 package org.elasticsearch.action.admin.indices.close;
 
+import org.elasticsearch.ElasticsearchException;
 import org.elasticsearch.Version;
+import org.elasticsearch.action.support.DefaultShardOperationFailedException;
 import org.elasticsearch.action.support.master.ShardsAcknowledgedResponse;
+import org.elasticsearch.common.Nullable;
+import org.elasticsearch.common.Strings;
 import org.elasticsearch.common.io.stream.StreamInput;
 import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.common.xcontent.ToXContentFragment;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.index.Index;
 
 import java.io.IOException;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.unmodifiableList;
 
 public class CloseIndexResponse extends ShardsAcknowledgedResponse {
 
+    private List<IndexResult> indices;
+
     CloseIndexResponse() {
     }
 
     public CloseIndexResponse(final boolean acknowledged, final boolean shardsAcknowledged) {
```

Comment:
nit: maybe we should remove this constructor to ensure it is no longer used? I personally prefer to let the caller explicitly pass the empty list, but this may be personal preference so feel free to omit.

Reply:
+1. I pushed https://github.com/elastic/elasticsearch/pull/39687/commits/1c9ddf0bc691a597e580148dcc6bb491280f1404.